packages/repository/src/repositories/legacy-juggler-bridge.ts

```diff
@@ -145,9 +144,19 @@ export class DefaultCrudRepository<
     const model = dataSource.getModel(definition.name);
     if (model) {
       // The backing persisted model has been already defined.
-      return model as typeof juggler.PersistedModel;
+      return model as typeof juggler.PersistedModel & ObserverMixin;
     }
 
+    return this.definePersistedModel(entityClass);
+  }
+
+  // Create an internal legacy Model attached to the datasource
```

Comment:
Please replace this comment with a proper tsdoc-style one, explain LB4 app developers that they can override this method in their custom repository classes to tweak the legacy Model (e.g. register operation hooks).

Reply:
👍